### PR TITLE
update snis to edgexfoundry.org;use kong-token.json token file in the…

### PR DESCRIPTION
**Pull request by tingyuz**
… configuration file

Signed-off-by: Tingyu Zeng <tingyu.zeng@rsa.com>

### CODE REVIEW

**Comment by anonymouse64** [comment]
recheck

File: core/res/configuration.toml [hunk 1]
```diff
@@ -40,9 +40,9 @@ server = "localhost"
 port = "8200"
 healthcheckpath = "v1/sys/health"
 certpath = "v1/secret/edgex/pki/tls/edgex-kong"
-tokenpath = "res\\resp-init.json"
+tokenpath = "res\\kong-token.json"
```
Comment (anonymouse64) [inline core/res/configuration.toml]: if you change this here we will need to make another change in edgex-go to fix the sed expression here: https://github.com/edgexfoundry/edgex-go/blob/edinburgh/snap/snapcraft.yaml#L977 so I'd prefer if you just change the configuration-docker.toml to unblock the docker deployment as the snap deployment is already fixed

**Review by anonymouse64 — CHANGES_REQUESTED** [review]
I'd rather not have the configuration.toml file changed, rather just the configuration-docker.toml file to prevent another PR to edgex-go

File: core/res/configuration.toml (hunk 1 above)
Comment (tingyuz) [reply core/res/configuration.toml]: no problem. i will switch it back.

**Review by anonymouse64 — APPROVED** [review]
Tested locally and I am able to use the CA from the docker container to access the proxy over HTTPS without any SSL errors:
```
$ curl --cacert EdgeXFoundryCA.pem --resolve edgex-kong:8443:127.0.0.1 https://edgex-kong:8443; echo
{"message":"no Route matched with those values"}
```

**Review by jpwhitemn — APPROVED** [review]

**Pull request merged**